os/android/iahwc2.h

```diff
@@ -308,8 +311,8 @@ class IAHWC2 : public hwc2_device_t {
   hwcomposer::GpuDevice device_;
   std::vector<std::unique_ptr<HwcDisplay>> extended_displays_;
   HwcDisplay primary_display_;
-  HwcDisplay virtual_display_;
-  HwcDisplay nested_display_;
+  std::vector<std::unique_ptr<HwcDisplay>> virtual_displays_;
+  uint32_t virtual_display_index_ = 0;
```

Comment:
Is this ever going to be anything other than virtual_displays_.size()?

Reply:
original code actually only support 1 VDS, here code is use for support multi VDS, virtual_displays_ is store the VDS, virtual_display_index is used identify the specified VDS.

Reply:
I'm looking for a case where these values diverge. It looks like they are both initialized to 0 here and then incremented at the same time in CreateVirtualDisplay, is there something I'm missing?

Reply:
why need values diverge here. when you create a new VDS, it init with a index, and store the instance to virtual_displays_. while call the display, it according different display_handle then retrieve from virtual_displays_ to make sure the call is assigned correct VDS instance. when you want destroy VDS, you can DestroyVirtualDisplay, it also accord the handle to retrieve the instance then remove from the virtual_displays_.